src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/ContaminationModel.java

```diff
@@ -160,15 +162,49 @@ public ContaminationModel(List<PileupSummary> sites) {
                 .mapToDouble(ps -> ps.getTotalCount() * oppositeAlleleFrequency.applyAsDouble(ps))
                 .sum();
 
-        final double contamination = contaminationOppositeDepth / totalDepthWeightedByOppositeFrequency;
+        final double contaminationEstimate = contaminationOppositeDepth / totalDepthWeightedByOppositeFrequency;
+
+        final double coeff1 = homs.stream().mapToDouble(ps -> oppositeAlleleFrequency.applyAsDouble(ps) * ps.getTotalCount()).sum();
```

Comment:
Could you specify what equation this is constructing by putting the tex in javadoc?

Reply:
done